scripts/source_code.py

```diff
@@ -130,18 +129,20 @@ def inspect_get_func_code_demangled(
         visitor = InclassFunctionFinder(class_name, function_name)
         visitor.visit(tree)
     # get the result
-    function_source_code = ""
-    for function_name, function_code in visitor.functions.items():
-        if function_code is not None:
-            function_source_code = function_source_code.join(function_code)
 
-    return function_source_code
+    function_source_code = "".join(
+        function_code
+        for function_code in visitor.functions.values()
+        if function_code is not None
```

Comment:
Just curious, can Python do something like `if not function_code: pass`?

Reply:
Yes, it can.